app/javascript/components/explore/DifferentialExpressionPanel.jsx

```diff
@@ -312,68 +332,91 @@ function DifferentialExpressionTable({
   return (
     <>
       <div className="de-table-buttons">
-        <DotPlotButton dotPlotGenes={dotPlotGenes} searchGenes={searchGenes} />
-        <DownloadButton bucketId={bucketId} deFilePath={deFilePath} />
+        {genesToShow.length > 0 &&
```

Comment:
Not a blocker:
I see `genesToShow.length` 4 times I think in this first section, I wonder if these could be pulled into one const? Also might be beneficial to add a check for `genesToShow` so maybe something like:
`const genesToShowLength = genesToShow?.length`